src/ui/console.c

```diff
@@ -2311,6 +2311,7 @@ cons_show_connection_prefs(void)
     cons_autoping_setting();
     cons_autoconnect_setting();
     cons_rooms_cache_setting();
+    cons_strophe_setting();
```

Comment:
Hehe Just wanted to write "I think this should go into `cons_show_connection_prefs()` or `cons_prefs()`" but after a refresh it was done already :)